src/components/SubmissionPage/SubmissionPage.jsx

```diff
@@ -0,0 +1,50 @@
+import { Flex, Input, Button } from "@chakra-ui/react";
+import { useRef, useState, useEffect } from "react";
+
+export const SubmissionPage = () => {
+  const [disableButton, setDisabledButton] = useState(true);
+  const inputRef = useRef();
+
+  function isUrlValid(string) {
+    var res = string.match(
+      /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g
+    );
+    return res !== null;
+  }
+
+  const onChangeHandler = (e) => {
```

Comment:
- for better readability, you can name this function something like `checkPortfolioUrl`
- this function isn't using event, so you don't need to take event as a parameter here 

```suggestion
  const checkPortfolioUrl = () => {
```

Reply:
Yes done